packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -133,12 +139,40 @@ contract CrossL2Inbox is ISemver, TransientReentrancyAware {
     /// @param _id      Identifier of the message.
     /// @param _msgHash Hash of the message payload to call target with.
     function validateMessage(Identifier calldata _id, bytes32 _msgHash) external {
```

Comment:
Since the `Identifier` type is being changed, can we rely on solidity fully now? In the spec I did add the `require` checks when it was still `uint256`, so that the Go implementation can safely use the shorter-typed identifier attributes.